### ISSUE TITLE
New lines stripped

### ISSUE THREAD

**Issue by ouranos** [SparkPost/mandrill-sparkpost-templates#9]
See this example:
```
{{#if newUser}}
  Welcome to
{{else}}
  Welcome back to
{{/if}}
our little block of code.
```

```
{{if newUser}}  Welcome to
{{else}}  Welcome back to
{{end}}our little block of code.
```

**Comment by ewandennis** [comment]
Thats interesting.  It looks like Handlebars is not preserving white space.  Thanks for the clean test case!

**Comment by ewandennis** [comment]
This is resolved in v0.1.4
